src/components/Routing.js

```diff
@@ -45,17 +49,24 @@ class Routing extends Component {
 
   render() {
     return (
+      <>
+      {this.state.isLoggedIn && <Navbar  changeLoginState = {this.updateLoginState}/>}
```

Comment:
NavBar always should be visible just change its content based on if a user is signed in or not.

Reply:
Done